gnocchi/tests/functional/gabbits/resource-type.yaml

```diff
@@ -769,7 +769,7 @@ tests:
             max_length: 255
       status: 400
       response_strings:
-       - "can't remove non-existent object 'what'"
+       - can't remove non-existent object 'what'
```

Comment:
The word `a` is missing from this message. See the test failure for the exact message, but I think it's supposed to be "can't remove _a_ non-existent..."

Reply:
eagle eyes.

Reply:
I'm a pterodactyl, caw caw.

Reply:
Says the _eye of lynx_.

http://eavesdrop.openstack.org/irclogs/%23openstack-telemetry/%23openstack-telemetry.2016-12-06.log.html#t2016-12-06T17:04:48

https://en.wiktionary.org/wiki/%C5%93il_de_lynx 😛

Reply:
/me looks at date of chat log

how is this possible? i can't remember yesterday.

Reply:
Wow, crazy how I could have missed that. Let's see if it works when fixing that. Thanks @cdent!